apps/accounts/models.py

```diff
@@ -179,6 +181,33 @@ class Meta:
     def __str__(self):
         return '{} - {} - {}'.format(self.user, self.song, self.emotion)
 
+    @classmethod
+    def get_attributes_for_emotion_and_context(cls, user_id, emotion_name, context):
```

Comment:
Delete this method once the browse playlist context logic has been updated to use the `average` function again